### PR TITLE
fix(Checkbox): Add border to disabled checkbox

### PR DESCRIPTION
**Pull request by gretanausedaite**
Added border to disabled checkbox for better visibility.

### CODE REVIEW

**Review by FlyersPh9 — APPROVED** [review]

File: src/inputs/checkbox.scss
Comment (mayank99) [inline src/inputs/checkbox.scss:89]: Do we want to exclude it from visibility checkbox? This looks a bit cluttered 
image

**Comment by FlyersPh9** [comment]
Wait for someone else's approval, I only gave a quick glance.  Might be able to apply the border in `checkbox-radio.scss` instead of only in `checkbox.scss` but I don't have time to review more thoroughly.

**Review by mayank99 — APPROVED** [review]

File: src/inputs/checkbox.scss
Comment (mayank99) [reply src/inputs/checkbox.scss]: Oh, you could avoid the duplicate variable declarations by using `:enabled` selector inside `:where` (as it does not increase specificity)

image

Comment (gretanausedaite) [reply src/inputs/checkbox.scss]: To me it would be a bit more clearer what colours are used in visibility checkbox when variables are duplicated. But it's not good to duplicate code. 
Weren't we removing `:enabled` selector from buttons when refactoring them?

Comment (mayank99) [reply src/inputs/checkbox.scss]: We were removing `:enabled` from button for two main reasons:
- specificity (which is solved by `:where`)
- polymorphic buttons (e.g. rendered as `<a>` which does not support `:disabled` or `:enabled`)

I'm fine with duplicate code here, it's only two lines.

**Review by bentleyvk — APPROVED** [review]

**Pull request merged**
